pages/components/Cards/cards.css

```diff
@@ -0,0 +1,316 @@
+    padding: 0 0 0 0.4rem;
+    color: var(--black);
+    margin-left: 0.4rem;
+    bottom: 8.6rem;
+    border-radius: 0.2rem;
+    z-index: 1;
+    padding-right: 0.3rem;
+}
+
+.product-count-seprator {
+    font-weight: bold;
+    padding-left: 0.4rem;
+}
+
+.like-box {
+    display: inline-block;
+    position: absolute;
+    top: 0.2rem;
+    right: 0.6rem;
+
+}
+
+.like-box:hover {
+    width: 4rem;
+    height: 4rem;
+    background-color: var(--light-red);
+    border-radius: 50%;
+
+}
+
+.like .icon {
+    font-size: 2rem;
+    color: var(--gold1);
+}
+
+.like .icon:hover {
+    color: var(--dark-red);
+}
+
+a {
+    text-decoration: none;
+    color: var(--gold1);
+}
+
+.product-type {
+    font-size: 1.7rem;
+    font-weight: bolder;
+}
+
+.product-name {
+    font-size: 1.4rem;
+    padding-right: none;
+}
+
+.product-price {
+    padding: 0 1rem 0 0;
+    text-align: center;
+    display: flex;
+}
+
+.product-discount-price {
+    font-size: 1.4rem;
+    font-weight: bolder;
+}
+
+.product-strike {
+    text-decoration: line-through;
+    font-size: 1.1rem;
+}
+
+.product-discount {
+    font-size: 1.5rem;
+    font-weight: bold;
+    padding-left: 0.5rem;
+    color: var(--tomatoOrange);
+
+}
+
+.product-meta-info {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: start;
+}
+
+/* card-variations */
+
+.prod-imgTag {
+    position: absolute;
+    top: 1rem;
+    background: rgba(204, 6, 29, 0.8);
+    padding: 0 0.6rem;
+    height: 2rem;
+    z-index: 3;
+    font-weight: 500;
+}
+
+.prod-imgTag::after {
+    content: "";
+    position: absolute;
+    left: 100%;
+    top: 0;
+    width: 0;
+    height: 0;
+    border-color: rgba(204, 6, 29, 0.5) transparent rgba(204, 6, 29, 0.5) rgba(204, 6, 29, 0.8);
+    border-width: 1rem;
+    border-style: solid;
+
+}
+
+.prod-imgTagBlur {
+    background: var(--greyLight-1);
+    top: 2rem;
+    position: absolute;
+    font-size: 1.6rem;
+    margin: 0 4rem;
+    width: 12rem;
+    color: var(--coralOrange);
+}
+
+.cross-box {
+    display: inline-block;
+    position: absolute;
+    top: 0.2rem;
+    right: 0.6rem;
+
+}
+
+.cross-box {
+    width: 4rem;
+    height: 4rem;
+    background-color: var(--blackl1);
+    border-radius: 50%;
+
+}
+
+
+.cross-icon {
+    font-size: 2.8rem;
+    width: 2rem;
+    height: 2rem;
+    color: var(--gold1);
+}
+
+.cross-box:hover {
+    background-color: var(--dullgold);
+}
+
+.card-btn {
+    padding: 0.8rem 1.6rem;
+    font-size: 1.8rem;
+    width: 100%;
+    color: var(--tomatoOrange);
+    border: 2px solid var(--blackl1);
+}
+
+.btn-primary {
+    padding: 0.8rem 1.4rem;
+    font-size: 1.7rem;
+    color: var(--gold1);
+    background-color: var(--dark-red1);
+    border-radius: 0rem;
+    width: 88%;
+    margin: 0.6rem 0;
+    text-transform: capitalize;
+    text-transform: uppercase;
+}
+
+
+.btn-primary:hover {
+    background:var(--dark-red2);
+    /* box-shadow: none; */
```

Comment:
delete the commented code before pushing

Reply:
okeiy i will refactor it later.